### DISCUSSION TITLE
media converter with module bay - cable trace?

### DISCUSSION THREAD

**Discussion by skydiablo** [netbox-community/netbox#9514] [Q&A] [answered]
i have got an media converter with an SFP slot on the one side and an simple copper port (RJ45) on the other side. how can i define this device? i would save the ability to trace a cable across the media-converter. if i add an module-bay for the SFP-slot i can plug an SFP-module, but this will provide an new interface (sure, i have defined it on SFP-Module) on the device (media converter). so how can i pass through this connection to my RJ45 Port and keep the ability for cable tracing across the media converter device?

**Comment by jeremystretch** [comment, marked as answer]
Module bays should not be used for optics; use inventory items if you need to track SFPs. You can model one side as a front port and the other as a rear port, to ensure the cable path is traced through the device.

**Reply by skydiablo** [reply]
so SFP-opticts should never implemented as a module? also for switches if the SFP-module act as an interface?
